### ISSUE TITLE
SVG's are all black in the preview

### ISSUE THREAD

**Issue by skippengs** [maphew/svg-explorer-extension#2]
Love this tool, but when al svg's are turned black it becomes useless.
Currently viewing a folder with svg's of country flags.
They all turn into the same generic shape and all detail is lost.
![2018-06-14 10_24_29-window](https://user-images.githubusercontent.com/7521903/41400349-309db876-6fbd-11e8-9968-cdda46b6e911.png)

**Comment by richard-scott** [comment]
See this [comment](https://github.com/maphew/svg-explorer-extension/issues/1#issuecomment-396395688), it may help.

**Comment by const-int** [comment]
Same problem here. Files exported from Adobe XD are black silhouettes.
![svg-demo](https://user-images.githubusercontent.com/5854245/42337678-c7eaeb74-8087-11e8-87a0-7391c66bf2a9.png)

**Comment by liftarn** [comment]
I have noticed that transparent backgrounds gets rendered as black, but you seem to have a different problem.

**Comment by roose** [comment]
It's because fill/stroke colors given by style not inline fill/stroke, by the way Figma can export it correctly

**Comment by aplato** [comment]
I had that problem as well. Fixed it by adding a [type] attribute with value of 'text/css'.

**Comment by PhilippSPOTTERON** [comment]
> I had that problem as well. Fixed it by adding a [type] attribute with value of 'text/css'.

where did you add this?

**Comment by aplato** [comment]
In the svg, I added the **type="text/css"**. Inkscape which i had used to generate the file omitted it.

```
<?xml version="1.0" encoding="UTF-8"?>
<svg version="1.1" viewBox="0 0 2263 2089" xmlns="http://www.w3.org/2000/svg" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
<style **type="text/css"**>.b {
   stroke-opacity:1;
   fill-opacity:1;
   ...
</style>
```

**Comment by thiemeljiri** [comment]
Hi,
I'm working on Windows 10 64bit and I've noticed that transparent background in SVG gets rendered as black. That's a problem as many SVGs are black and transparent, e.g. icons. I strongly believe that there are lots of us working mainly with icons which are black and transparent and fixing this issue would really make a lot of the users much happier.

How about render the SVG on gray and white checkboard as people are used to? The preview would never be transparent this way and the problem should disappear. If the issue with alpha channel rendered as black happens only to some users, ading this as an option which can be turned on/off would be amazing!

Thanks for your work!

Cheers,
Jiri

**Comment by neoteknic** [comment]
type="text/css" is working, so can you fix this bug ( style with no type <style> = defaut to type=text/css). I think that it's easy to fix

**Comment by borisovmi** [comment]
I have the same problem after renaming the file

**Comment by maphew** [comment]
Closing, as is this is fixed in #32. Please note this means that the _code_ is fixed. It will still be an indefinite time until the installer is updated to include the fix (because I haven't learned how yet).
